apps/extension/src/ui/domains/Sign/Ethereum/EthSignBodyMessageSIWE.tsx

```diff
@@ -0,0 +1,119 @@
+import { AccountJsonAny } from "@core/domains/accounts/types"
+import { EthSignRequest } from "@core/domains/signing/types"
+import { log } from "@core/log"
+import { hexToString } from "@polkadot/util"
+import { ParsedMessage } from "@spruceid/siwe-parser"
+import { UserRightIcon } from "@talismn/icons"
+import { useEvmNetwork } from "@ui/hooks/useEvmNetwork"
+import { FC, useMemo } from "react"
+import { useTranslation } from "react-i18next"
+import { Button, Drawer, useOpenClose } from "talisman-ui"
+
+import { SignAlertMessage } from "../SignAlertMessage"
+import { ViewDetailsAddress } from "../ViewDetails/ViewDetailsAddress"
+import { ViewDetailsButton } from "../ViewDetails/ViewDetailsButton"
+import { ViewDetailsField } from "../ViewDetails/ViewDetailsField"
+import { SignParamAccountButton } from "./shared"
+
+const ViewDetailsContent: FC<{
+  account: AccountJsonAny
+  request: EthSignRequest
+  siwe: ParsedMessage
+  onClose: () => void
+}> = ({ account, request, siwe, onClose }) => {
+  const { t } = useTranslation("request")
+  const evmNetwork = useEvmNetwork(String(siwe.chainId))
+
+  const message = useMemo(() => hexToString(request.request), [request.request])
+
+  return (
+    <div className="bg-grey-850 flex max-h-[60rem] w-full flex-col gap-12 p-12">
+      <div className="scrollable scrollable-700 flex-grow overflow-y-auto overflow-x-hidden pr-4 text-sm leading-[2rem]">
+        <div className="text-body-secondary">{t("Details")}</div>
+        <p>
+          {t(
+            "You are about to authenticate via Ethereum. Please ensure you trust the application before continuing."
```

Comment:
Maybe we should say `sign in with Ethereum` rather than `authenticate` since the phrase is a recognisable "brand" that has been marketed as part of the EIP?